docs/upgrade.md

```diff
@@ -55,6 +55,10 @@ Bubbled child events no longer pass the `childView` implicitly and only pass the
 arguments passed as part of `triggerMethod`. This means that the arguments
 passed to `onEvent` and `onChildviewEvent` are now identical.
 
+In Marionette 2, `childEvents` where bound on every event. In Marionette 3,
+`childViewEvents` are bound once and cached. This means that you cannot add new
```

Comment:
should
> This means that you cannot add new events after the view has been created.

be
> This means that the parent views cannot listen to new childViewEvents after the view has been created.

Since thats the case, is there a way for someone to bind and cache them themselves?